app/src/main/java/com/example/playlistmaker/SearchActivity.kt

```diff
@@ -70,17 +93,79 @@ class SearchActivity : AppCompatActivity() {
             }
 
         }
+
+        fun sendToServer() {
+            if (searchField.text.isNotEmpty()) {
+                iTunesService.search(searchField.text.toString())
+                    .enqueue(object : Callback<TrackResponse> {
+                        @SuppressLint("NotifyDataSetChanged")
+                        override fun onResponse(
+                            call: Call<TrackResponse>,
+                            response: Response<TrackResponse>
+                        ) {
+                            rwTrack.isVisible = true
+                            holderNothingOrWrong.isVisible = false
+
+                            if (response.code() == 200) {
```

Comment:
Чтобы не хардкодить в активити значения http кодов, можно воспользоваться полем `isSuccessfull`:
```suggestion
                            if (response.isSuccessfull) {
```